### PR TITLE
Add a setting to disable DPI scaling on Windows systems

### LINKED ISSUE #13099: Add a setting to disable DPI scaling
See https://github.com/OpenRA/OpenRA/pull/12501#issuecomment-292999888, #13097 and [playtest-20170408/#comment-3250354354](http://www.openra.net/news/playtest-20170408/#comment-3250354354).

### PR DESCRIPTION
**Pull request by obrakmann**
Closes #13099.

### CODE REVIEW

File: OpenRA.Platforms.Default/Sdl2GraphicsDevice.cs [hunk 1]
```diff
@@ -79,7 +79,7 @@ public Sdl2GraphicsDevice(Size windowSize, WindowMode windowMode)
 				SurfaceSize = new Size(width, height);
 				WindowScale = width * 1f / WindowSize.Width;
 			}
-			else if (Platform.CurrentPlatform == PlatformType.Windows)
+			else if (Platform.CurrentPlatform == PlatformType.Windows && !Game.Settings.Graphics.DisableWindowsDPIScaling)
```
Comment (pchote) [inline OpenRA.Platforms.Default/Sdl2GraphicsDevice.cs]: You'll need to disable the `SetProcessDPIAware` above too.

**Review by pchote — CHANGES_REQUESTED** [review]
Should be good after this:

**Comment by obrakmann** [comment]
Updated.

**Review by reaperrr — APPROVED** [review]

**Review by pchote — APPROVED** [review]
:+1: does the job as advertised.

Two points worth noting:
* When this setting is enabled and Window's built in scaling is used cursors will not be scaled.  This is consistent with all our releases before HiDPI support was added.
* Windows's "Disable DPI scaling on high DPI displays" compatability option only acts on the system provided scaling.  The setting in this PR must be enabled for that checkbox to work as expected.

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/bf00831cc347c6df09d521b4c884560edae069b7 and pushed to prep-1703.

**Pull request merged**